### DISCUSSION TITLE
umi初始化的项目的入口文件在哪

### DISCUSSION THREAD

**Discussion by Xiongyonglong** [umijs/umi#7540] [Q&A]
我想在ReactDOM.render(<App />, document.querySelector('#root')) 挂载的地方做一些判断修改，但是没找到入口文件是哪个，或者说没找到ReactDOM.render(<App />, document.querySelector('#root')) 在哪

**Comment by Xiongyonglong** [comment]
求教各位大佬

**Reply by Xiongyonglong** [reply]
版本是umi2.9

**Reply by puxiao** [reply]
我用的是最新版 3.5.x，也是刚学没多久。

路由是 umi (也是 antd pro) 的一切开始与核心。
create-react-app 默认显示的内容主要依赖的是 首页组件(/src/app.jsx) 中所填充的具体内容
而 umi（antd pro） 默认显示的内容主要依赖的是 “此时此刻的路由(url 网址)”

对于 umi 而言，它的静态页模板文件是：src/pages/document.ejs
同时你看一下 src/layouts/BasicLayout.jsx 这个文件，有可能是你想要找的。
